internal/fs/rename_go1.7.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016 The Go Authors. All rights reserved.
```

Comment:
In `internal/gps`, we have files named `internal/gps/remove_go17.go` and `internal/gps/remove_go16.go`. I think it would be best to match that style, so this should be called `internal/fs/rename_go17.go`.